src/styles/components/service-table/styles.less

```diff
@@ -43,10 +43,6 @@
       }
     }
 
-    .service-status-icon-wrapper {
-      width: 110px;
```

Comment:
This was causing a visual bug with the progress bar. Since I couldn't figure out a good reason for having it, I am deleting it.
![Screenshot from 2019-05-27 11-15-29](https://user-images.githubusercontent.com/40791275/58416815-84e63680-808b-11e9-938e-c55fe16b60ca.png)